Tests/Shared/Reachability/NetworkingReachabilityManagerTests.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  NetworkingReachabilityManagerTests.swift
+//  VimeoNetworking
+//
+//  Created by Rogerio de Paula Assis on 8/29/19.
+//  Copyright © 2019 Vimeo. All rights reserved.
+//
+
+import XCTest
+@testable import VimeoNetworking
+
+class NetworkingReachabilityManagerTests: XCTestCase {
+
+    func testThatManagerCanBeInitializedFromHost() {
```

Comment:
The test methods here don't seem to follow our format in the style [guide](https://github.vimeows.com/MobileApps/Style-Guides/tree/master/Apple#naming-unit-tests).